controllers/courses.js

```diff
@@ -80,6 +80,7 @@ const getSyncedElements = 	(
 		startDate,
 		untilDate,
 		syncedWithGroup,
+		syncExcludedFields: course.syncExcludedFields?.join(','),
```

Comment:
i don't like the var name 😅 I also don't have a good one... But syncSomething sounds like this should be synced though we want the exact opposite! It should be more like excludeFieldsForSync (🤔) .. 🤷‍♂️